range.go

```diff
@@ -51,6 +52,11 @@ type Range struct {
 	dur time.Duration
 }
 
+type marshalTime struct {
+	StartTime time.Time `json:"StartTIme"`
```

Comment:
I would prefer to use snake case keys for json values, e.g. "start_time" and "end_time". Also, I'm not sure that the tests will pass, as this key has "I" letter in the upper case, when in the test it is in the lower case